evm/verify-contract.js

```diff
@@ -33,26 +33,22 @@ async function processCommand(config, chain, options) {
     printInfo('Contract address', options.address || chain.contracts[contractName]?.address);
 
     const contractConfig = chain.contracts[contractName];
+    const contractJson = getContractJSON(contractName);
+    let contract;
+    const contractFactory = await getContractFactoryFromArtifact(contractJson, wallet);
+    const excludedContracts = ['InterchainProposalSender', 'BurnableMintableCappedERC20', 'TokenManagerProxy', 'AxelarAmplifierGateway'];
+
+    if (!excludedContracts.includes(contractName)) {
+        contract = contractFactory.attach(options.address || contractConfig.address);
```

Comment:
```suggestion
        const contractAddress = options.address || contractConfig.address;
```
instead create a var, and create contract object only for the cases where necessary